pkgs/by-name/oi/oink-solver/package.nix

```diff
@@ -0,0 +1,49 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  cmake,
+  lace,
+  boost,
+  build-extra-tools ? false,
+  doCheck ? false,
```

Comment:
Why are they `false` by default?

Reply:
The extra tools are false by default, mirroring the upstream repo's default. I disabled the checks by default as they use a lot of memory (my 16GiB machine instantly ran out of memory).

Reply:
Please leave a comment about the memory usage for `doCheck` :-)